src/components/Chip.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { Box as BoxPrimitive, BoxProps as BoxPrimitiveProps } from 'mdlz-prmtz';
+import { variant, Prop } from '@modulz/radix-system';
+import { setLightness } from 'polished';
+import {theme} from '../theme';
+
+type ChipProps = BoxPrimitiveProps & {
+  variant?: Prop<'gray' | 'primary' | 'blue' | 'green' | 'purple' | 'orange'>;
+  size?: Prop<0 | 1 | 2>;
+  truncate?: Prop<boolean>;
+};
+
+export const Chip = React.forwardRef<HTMLDivElement, ChipProps>((props, ref) => (
+  <BoxPrimitive
+    {...props}
+    ref={ref}
+    css={[
+      css({
+        fontWeight: 600,
+        margin: 1,
```

Comment:
Components shouldn't outer spacing so margin shoud be removed. The developer will still have the choice to add a margin using the component (ex: ```<Chip margin={1} >```)